manim/mobject/types/vectorized_mobject.py

```diff
@@ -843,6 +844,51 @@ def get_subcurve(self, a, b):
         vmob.pointwise_become_partial(self, a, b)
         return vmob
 
+    """
+    The next 3 methods are for getting or changing the direction of
+    points in 2D. The direction can either be "CCW", counterclockwise,
+    or "CW", clockwise. The default is counterclockwise. This also
+    determines how the objects are drawn which can by changed by
+    reverse_direction or force_direction. This can also be used to make
+    operations on objects that are dependent on direction much easier.
+    """
+
+    def get_direction(self):
+        """
+        Uses shoelace in space_ops.py to calculate the direction.
+
+        Returns
+        ------
+        str ("CW" or "CCW")
+        """
+        return shoelace(self.get_start_anchors(), False, True)
+
+    def reverse_direction(self):
```

Comment:
This can just be `self.points = self.points[::-1]`.

Reply:
Good catch, I wrote that when I had less clue about direct attribute access and didn't really connect the dots while copying it over. But it does also raise the point of get_points being the exact kind of pointless get function we should get rid of.